test/focus.jest.js

```diff
@@ -44,7 +44,7 @@ describe('Page.focus', function() {
     expect(focused).toBe(true);
     expect(blurred).toBe(true);
   });
-  it.fail(WEBKIT && !LINUX)('should traverse focus', async function({page, server}) {
```

Comment:
I tested this on my Mac and windows and it worked fine, so I am not sure why it was marked failing. Maybe the bots will tell a different story.